src/UsersList.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import gql from "graphql-tag";
+import { graphql } from "react-apollo";
+
+const UsersList = ({ users }) => {
+  return (
+    <div>
+      {users.map(user =>
+        <div key={user.id}>
+          {user.email}
+        </div>
+      )}
+    </div>
+  );
+};
+
+UsersList.defaultProps = {
+  users: []
+};
+
+const allUsersQuery = gql`
+  query users {
+    allUsers {
```

Comment:
Is there a way to filter by when users joined? Do we even store that data?

Reply:
Yeah, we have a createdAt value. Looks like graph.cool will allow us to sort by this date or we can just create a query to grab only the people who signed up on that day. Let me know which you would like to do. @wdoug